### PR TITLE
4-GET api/reviews

### PR DESCRIPTION
**Pull request by malik9153**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hello :) 

Please...
-Add a test for the order pf the reviews (using jest-sorted)
-If you are using a string literal to add DESC to your db.query, please use an if statement to make sure that it is valid first (to avoid SQL injection).

Please send another PR when you have taken a look at these and we will check again :) Thanks.

File: __tests__/app.test.js [hunk 1]
```diff
@@ -26,4 +26,37 @@ describe("GET /api/categories", () => {
...
+      .expect(200)
+      .then(({body}) => {
+        body.forEach((category) => {
+          expect(category).toMatchObject({
+            review_id:expect.any(Number),
+            title:expect.any(String),
+            designer:expect.any(String),
+            owner:expect.any(String),
+            review_img_url:expect.any(String),
+            review_body:expect.any(String),
```
Comment (hevdevs) [inline __tests__/app.test.js:45]: we **don't** want to be selecting the body from all reviews here

File: controller/controller.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
     next(err);
-   
-});
+  });
+}
+
+exports.getReviews = (req, res, next) => {
+    SelectReviews()  
+    .then((response) => {
+        res.status(200).send(response);
```
Comment (hevdevs) [inline controller/controller.js:17]: we **always** want to be sending our response back as an object with a descriptive key - i.e. here you'd want something like .send({reviews: response})

File: controller/controller.js [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
+      })
+      .catch((err) => {
+        console.log(err, "controller reject");
```
Comment (hevdevs) [inline controller/controller.js:20]: remove all console.logs before merge

**Review by hevdevs — CHANGES_REQUESTED** [review]
Couple of bits to change here Malik but looking good overall! 
// afterAll(() => {
//   db.end();
// });
 ^ You need this in your tests to stop jest from hanging, all you need to do is make sure you are returning db.end() to avoid errors from multiple connections running at once.
 It's also a good idea to name your models and controllers after the endpoints/tables they concern instead of having them all in one big file i.e. categories.controller/model, reviews.controller/model, etc. This is to make your code easier to navigate and for separation of concerns given that this is a pretty big project! 
At this point I would also advise putting in an error test for a generic 404 path not found when the url is anything other than what we have specified as valid in our app.js. 
 Please throw us another PR once this is done

**Pull request merged**